supervisor/dbus/const.py

```diff
@@ -198,6 +198,13 @@ class InterfaceMethod(str, Enum):
     LINK_LOCAL = "link-local"
 
 
+class InterfaceAddrGenMode(int, Enum):
```

Comment:
```suggestion
class InterfaceAddrGenMode(IntEnum):
```
Also will be more obvious after a rebase as all the others are changed this way. Inheriting from `IntEnum` is the new improved alternative to inheriting from `int` and `Enum` as of python 3.11.